### PR TITLE
Add migrate Analytics to Edge Network doc

### PR DESCRIPTION
**Pull request by cacheung**
- Add send Analytics to Edge Network migration doc under Analytics extension workspace
- Add how to send Analytics to React Native or Flutter under Analytics faq
- Add an Identity question under Edge Identity faq

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/adobe-analytics/migrate-to-edge-network.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+| 2. [Configure a datastream](https://experienceleague.adobe.com/docs/experience-platform/edge/datastreams/overview.html). A datastream is the server-side configuration used when implementing the Experience Platform Mobile SDK. | ✅	 | ✅	|
+| 3. **Add the Adobe Analytics service** to your datastream. The datastream controls both whether and how data is sent to Adobe Analytics. You will need your Analytics report suite ID (RSID) for this step.  | ✅ | ✅ |
+| 4. **Install the *Edge Network* and *Identity for Edge Network extensions*** in the mobile property (tag) in Data Collection UI, and set the datastream in the Edge Network extension configuration. |  ✅ | ✅ |
+| 5. **In your app code**, register the **Edge Network** and **Identity for Edge Network extensions** and load your mobile tag property configuration. | ✅ | ✅ |
```
Comment (emdobrin) [inline src/pages/documentation/adobe-analytics/migrate-to-edge-network.md:45]: load -> set

File: src/pages/documentation/adobe-analytics/migrate-to-edge-network.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+### Consent for Edge Network extension
+The Consent for Edge Network mobile extension enables consent preference collection from your mobile app when using the Experience Platform Mobile SDK and Edge Network.
+
+The privacy status setting (MobilePrivacyStatus.OPT_IN, OPT_OUT, UNKNOWN) set in the mobile tag property which is used by Adobe Experience Cloud mobile extensions (for example, Analytics and Target), are not read nor used by the Edge Network extensions. The Edge Network extensions rely on the Consent for Edge Network extension for this setting.
+
+For more details, please refer to the [Consent for Edge Network extension documentation](https://developer.adobe.com/client-sdks/documentation/consent-for-edge-network).
+
+
+### Identity consideration for the migration
+When using Identity for Edge Network extension, the Mobile SDK automatically migrates any existing MID/ECID to the new XDM IdentityMap format out of the box. If you are using custom identities with `syncIdentifier` / `syncIdentifiers` APIs, you should consider migrating your IDs to the XDM format.
```
Comment (emdobrin) [inline src/pages/documentation/adobe-analytics/migrate-to-edge-network.md:79]: - any existing MID/ECID -> an existing ECID (previously known as MID) 
- custom identities with **the** ...

**Review by emdobrin — COMMENTED** [review]
looks great

File: gatsby-config.js [hunk 3]
```diff
@@ -406,6 +406,10 @@ module.exports = {
                             {
                                 title: 'Release notes',
                                 path: '/documentation/adobe-analytics/release-notes/'
+                            },
+                            {
+                                title: 'Migrate to Edge Network',
+                                path: '/documentation/adobe-analytics/migrate-to-edge-network'
```
[the remark this answers is not in the document]
Comment (cacheung) [reply gatsby-config.js]: Thanks for the comments.  Updated!

File: src/pages/documentation/adobe-analytics/migrate-to-edge-network.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+If you are not using the latest version of the Mobile SDK (that is, still on v4 Mobile SDK or ACP-prefix Mobile SDK), we recommend that you first migrate to the [latest version](https://developer.adobe.com/client-sdks/documentation/current-sdk-versions), and then continue with this guide.
+
+### Consent for Edge Network extension
+The Consent for Edge Network mobile extension enables consent preference collection from your mobile app when using the Experience Platform Mobile SDK and Edge Network.
+
+The privacy status setting (MobilePrivacyStatus.OPT_IN, OPT_OUT, UNKNOWN) set in the mobile tag property which is used by Adobe Experience Cloud mobile extensions (for example, Analytics and Target), are not read nor used by the Edge Network extensions. The Edge Network extensions rely on the Consent for Edge Network extension for this setting.
+
+For more details, please refer to the [Consent for Edge Network extension documentation](https://developer.adobe.com/client-sdks/documentation/consent-for-edge-network).
+
+
```
Comment (calebk1) [inline src/pages/documentation/adobe-analytics/migrate-to-edge-network.md:77]: Extra newline here.

File: src/pages/documentation/adobe-analytics/migrate-to-edge-network.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+### Identity consideration for the migration
+When using Identity for Edge Network extension, the Mobile SDK automatically migrates an existing ECID (previously known as MID) to the new XDM IdentityMap format out of the box. If you are using custom identities with the `syncIdentifier` / `syncIdentifiers` APIs, you should consider migrating your IDs to the XDM format.
```
Comment (calebk1) [inline src/pages/documentation/adobe-analytics/migrate-to-edge-network.md:79]: Missing newline here.

File: src/pages/documentation/adobe-analytics/migrate-to-edge-network.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+
+The previous Identity for Experience Cloud Id Service extension can be safely removed from the SDK registration process if there are no other extensions which rely on it, such as Target, Campaign and others.
+
+If you need further assistance, please contact [Adobe Experience Cloud customer care](https://experienceleague.adobe.com/?support-solution=General#support).
+
+
+
+
+
```
Comment (calebk1) [inline src/pages/documentation/adobe-analytics/migrate-to-edge-network.md:88]: Don't need so many newlines here either.

File: src/pages/documentation/identity-for-edge-network/faq.md [hunk 7]
```diff
@@ -127,3 +127,9 @@ Android
 iOS
 
 <Tabs query="platform=ios&task=link"/>
+
+## Q: Can I safely remove the Identity for Experience Cloud Id Service extension in an app if I am using the Edge Network extension?
+
+If no other extension relies on the Identity for Experience Cloud Id Service extension, you can safely remove it. Please refer to the [Identity consideration document](../adobe-analytics/migrate-to-edge-network.md#identity-consideration-for-the-migration) for more information.
```
Comment (calebk1) [inline src/pages/documentation/identity-for-edge-network/faq.md:133]: Is it "ID service extension"? (Just seeing Id written like that slightly bugs me, that's why.)

**Review by calebk1 — COMMENTED** [review]
Looks good! Just a few minor blurbs.

File: src/pages/documentation/adobe-analytics/faq.md [hunk 8]
```diff
@@ -58,3 +58,8 @@ In the mobile property (tag) in the Data Collection UI, select the property from
 ### Contact Adobe Customer Care <a id="contact-adobe-customer-care"></a>
 
 If you are unable to resolve your concerns through resources provided here, please contact [Adobe Experience Cloud customer care](https://experienceleague.adobe.com/?support-solution=General#support) for immediate assistance.
+
+
+## How do I send data to Adobe Analytics in React Native and Flutter?
+
+In the latest SDK version (AEP-prefixed) for React Native and Flutter you can send data to Analytics via the Edge Network using the Edge Network extension and `sendEvent` API. If you want a drop-in migration solution for an existing `trackAction` / `trackState` implementation, you can do this with the help of the Edge Bridge extension. You can learn more about the options in the [migration guide](../adobe-analytics/migrate-to-edge-network.md).
```
[the remark this answers is not in the document]
Comment (cacheung) [reply src/pages/documentation/adobe-analytics/faq.md]: Thank you!  Updated.

File: src/pages/documentation/adobe-analytics/migrate-to-edge-network.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+The privacy status setting (MobilePrivacyStatus.OPT_IN, OPT_OUT, UNKNOWN) set in the mobile tag property which is used by Adobe Experience Cloud mobile extensions (for example, Analytics and Target), are not read nor used by the Edge Network extensions. The Edge Network extensions rely on the Consent for Edge Network extension for this setting.
+
+For more details, please refer to the [Consent for Edge Network extension documentation](https://developer.adobe.com/client-sdks/documentation/consent-for-edge-network).
+
+### Identity consideration for the migration
+
+When using Identity for Edge Network extension, the Mobile SDK automatically migrates an existing ECID (previously known as MID) to the new XDM IdentityMap format out of the box. If you are using custom identities with the `syncIdentifier` / `syncIdentifiers` APIs, you should consider migrating your IDs to the XDM format.
+
+The previous Identity for Experience Cloud Id Service extension can be safely removed from the SDK registration process if there are no other extensions which rely on it, such as Target, Campaign and others.
```
Comment (calebk1) [inline src/pages/documentation/adobe-analytics/migrate-to-edge-network.md:84]: "... Experience Cloud ID Service extension can be..."

**Review by calebk1 — APPROVED** [review]
LGTM! Just one small comment.

**Pull request merged**